src/components/Episodes/renderUtils.jsx

```diff
@@ -4,25 +4,28 @@ import Episode from "COMPONENTS/Episode/";
 function renderEpisode(data, index){
 
 	const { 
-		number,
+		episodeNo,
 		title,
 		synopsis,
-		image,
-		video,
+		thumbnail,
+		trailer,
 	} = data;
 
-	const id = `episode__${number}`;
+	const id = title
+			.replace(/\W+/g, '-') // remove non-alphanumeric & replace with -s
+			.toLowerCase()
+			;
```

Comment:
Use the same util we mentioned in the other component

Reply:
Created convertToHyphenatedString in utils.js, now being imported and used. 
Name is lengthy but descriptive :)